extensions/SampleExtension/SampleExtension.cs

```diff
@@ -4,23 +4,36 @@
 using System;
 using System.Runtime.InteropServices;
 using System.Threading;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
 using Microsoft.Windows.DevHome.SDK;
+using SampleExtension.Providers;
+using Serilog;
 
 namespace SampleExtension;
 
 [ComVisible(true)]
+#if CANARY_BUILD
```

Comment:
Does canary/stable make sense for a public sample? In theory shouldn't the sample be as easy to follow as possible and more complex scenarios can be gleamed from the rest of the dev home extension code?

Reply:
The intent is to mirror the production extensions and developers may want to have developer vs release versions of their extensions. One problem we had recently resolved is extensions with the same GUIDs/package names were colliding and causing problems. I think it is worthwhile to show a good example of how to set up the project such that you can have different versions co-existing on a system without stomping on each other.  Our production extensions do this, so I think it's a good practice to show our sample extension doing it as well. 

We also plan to incorporate the sample extension into test builds and automated UI testing, so having it built similarly to Dev Home and other extensions will allow those tests to function better. It's a bit of future proofing and also demonstrating a good way to have dev/non-dev versions of your extension.